tests/skipped_tests_gpu_no_fp64.tbl

```diff
@@ -409,7 +409,6 @@ tests/third_party/cupy/math_tests/test_sumprod.py::TestNansumNanprodLong_param_3
 tests/third_party/cupy/math_tests/test_sumprod.py::TestDiff::test_diff_2dim_with_scalar_append
 
 tests/third_party/cupy/math_tests/test_trigonometric.py::TestTrigonometric::test_deg2rad
-tests/third_party/cupy/math_tests/test_trigonometric.py::TestTrigonometric::test_hypot
```

Comment:
`@pytest.mark.usefixtures("allow_fall_back_on_numpy")` decorator can be removed for `test_hypot` in `tests/test_mathematical.py`